### ISSUE TITLE
Incompatibility between papertweaks with regions UI

### ISSUE THREAD

**Issue by bradgillap** [oddlama/vane#236]
Hello,

- Papermc Build #401
- **PaperTweaks.jar 0.4.1** https://hangar.papermc.io/Machine_Maker/PaperTweaks
- vane-regions-1.14.0.jar

I've identified that papertweaks causes some strange UI compatibility issues with regions on vane. I think regions still works but the menus display really weird. 

![image](https://github.com/oddlama/vane/assets/1094081/17e3fac7-ebc5-452d-bdeb-9e17f07ce048)

I haven't identified if it's a single papertweaks module or if it's just the entire plugin at this time.  I'll just remove paper tweaks as a solution for myself but I didn't see this documented anywhere so far in my google searches.

Anyway, hope this helps someone.  Vane is really cool, thanks for your hard work.

**Comment by oddlama** [comment]
This looks like paper tweaks modifies the lore of that item and in doing so they strip the special "translated text components" from the lore, so your client doesn't know that it has to look up a translation anymore.

If the problem really goes away when removing papertweaks, then it's likely that they have to make a fix for this.

**Comment by Machine-Maker** [comment]
To add some clarity here, I'm guessing this is just a result of conflicting resource packs? I haven't yet adopted the new pack behavior where clients can have multiple applied from the server. So if you have a module in PaperTweaks enabled which sends a resource pack, it replaces the pack from vane and so you are missing translations.

I will update my plugin to use the new methods to not remove all packs first, I think vane should do the same (tho I think PaperTweaks should load after vane, probably good to update it anyways).

**Comment by oddlama** [comment]
>  So if you have a module in PaperTweaks enabled which sends a resource pack, it replaces the pack from vane and so you are missing translations.

That makes sense, thanks for figuring it out!

> where clients can have multiple applied from the server

Oh my, this exists now? Finally! Do you happen to know when this was added? Anyway, I'll definitely have to switch to that, been waiting for this forever. Vane can probably be quite low-priority in the chain of packs.

**Comment by Machine-Maker** [comment]
> Oh my, this exists now? Finally! Do you happen to know when this was added?

Yeah, new functionality was added in the 1.20.3/4 update for this. There are new methods in the API `sendResourcePacks` on adventure's Audience interface for this.
